op-service/sources/engine_client.go

```diff
@@ -58,7 +58,17 @@ func (s *EngineClient) ForkchoiceUpdate(ctx context.Context, fc *eth.ForkchoiceS
 	fcCtx, cancel := context.WithTimeout(ctx, time.Second*5)
 	defer cancel()
 	var result eth.ForkchoiceUpdatedResult
-	err := s.client.CallContext(fcCtx, &result, "engine_forkchoiceUpdatedV3", fc, attributes)
+	method := "engine_forkchoiceUpdatedV3"
```

Comment:
Can we pull this out into it's own function?

Reply:
https://github.com/ethereum-optimism/optimism/pull/9253/commits/51f18b03afd5ebe5bdc2c01dfb685cb798e6776b: I have moved those checks into rollup config methods.